src/services/anchor/index.ts

```diff
@@ -210,16 +227,18 @@ export async function sep24First(
   const { token, tomlValues } = sessionParams;
   const { sep24Url } = tomlValues;
 
+  const { usesMemo } = OUTPUT_TOKEN_CONFIG[outputToken];
+
   let sep24Params;
-  if (outputToken == 'ars') {
-    if (!sep10Account) {
-      throw new Error('Master must be defined at this point.');
-    }
+  if (usesMemo) {
     sep24Params = new URLSearchParams({
       asset_code: sessionParams.tokenConfig.stellarAsset.code.stringStellar,
       amount: sessionParams.offrampAmount,
-      account: sep10Account, // THIS is a particularity of Anclap. Should be able to work without it, or with a different one
-      // to that of the sep-10
+      account: sep10Account, // THIS is a particularity of Anclap. Should be able to work just with the epmhemeral account
```

Comment:
We are still waiting for Anclap to solve on their side, or tell us if we are sending something wrong. 
The commented `memo` parameter is deprecated... but it could be the case that will be required.